raiden_contracts/tests/fixtures/token_network_registry.py

```diff
@@ -1,7 +1,8 @@
-from typing import Callable
+from typing import Any, Callable, Collection, List
 
 import pytest
 from eth_utils import is_address
+from web3 import Web3
```

Comment:
I expected this and the other `Web3` and `Contract` imports to be in an `if TYPE_CHECKING:` block. Is the pattern not used in `raiden_contracts`?

Reply:
No, I've never seen the pattern here.  Probably because in most cases we need the library at runtime.